### DISCUSSION TITLE
macbook touchbar

### DISCUSSION THREAD

**Discussion by jimmy0699** [scandum/tintin#33] [Ideas]
Is any one think of support for macbook touchbar ie. add custom actions that shows up when needed in touchbar like kill monster, drink, read letter etc

**Comment by scandum** [comment]
I think iTerm2 does something with the touch bar, is that what you're looking for? 

https://iterm2.com/documentation-touch-bar.html

**Reply by jimmy0699** [reply]
well, yes something in this direction, but more dynamically, ie:

```
#action {You killed monster.} {
   #touchbar {loot body} {take money from body} {10}
}
```

so, when action `You killed monster.` been trigger new touchbar button appear with label `loot body` when pressed `take money from body` will be sent to mud, available for 10s.

**Reply by scandum** [reply]
You'd be better off adding clickable substitutions.

See #help MSLP
